modules/ROOT/pages/6.0-release-notes-core-changes.adoc

```diff
@@ -0,0 +1,297 @@
+
+| `noneditable_noneditable_class`     | `noneditable_class`              |                                     | After upgrading, rename the options in your {productname} init configuration to match the new name. For example, `noneditable_noneditable_class: 'mceNonEditable'` must be renamed `noneditable_class: 'mceNonEditable'`.                                                                                     | see xref:changed-plugins-noneditable[Noneditable]
+
+| `noneditable_editable_class`        | `editable_class`                 |                                     | After upgrading, rename the options in your {productname} init configuration to match the new name. For example, `noneditable_editable_class: 'mceEditable'` must be renamed `editable_class: 'mceEditable'`.                                                                                                 | see xref:changed-plugins-noneditable[Noneditable]
+
+| `styleselect`                       | `styles`                         | Toolbar item                        | No behavioral changes.                                                                                                                                                                                                                                                                                       |
+
+| `formats`                           | `styles`                         | Menu item                           | No behavioral changes.                                                                                                                                                                                                                                                                                       |
+
+| `textpattern_patterns`              | `text_patterns`                  | Option                              | After upgrading, rename the options in your {productname} init configuration to match the new name. Also, remove `textpattern` from your plugins list. This name-change is consequent to `textpattern` being changed from a Plugin to being part of the {productname} Core.                                   | see xref:changed-plugins-textpattern[Textpattern]
+
+| `tinymce.Env.browser.isChrome`      | `tinymce.Env.browser.isChromium` | API                                 | Updated so the `Sand` and `Env` APIs better reflect what they are checking for. `isChrome` implies they are checking for _Google Chrome_. They are actually checking for any Chromium-based browser (eg Chromium, Google Chrome, or Chrome Edge) so `isChromium` more accurately reflects what is being done. |
```

Comment:
`Sand` is an internal only library so this might confuse any customers who read this?
```suggestion
| `tinymce.Env.browser.isChrome`      | `tinymce.Env.browser.isChromium` | API                                 | Updated so the `Env` API better reflects what is being checked. `isChrome` implies it was checking for _Google Chrome_. Instead, it was actually checking for any Chromium-based browser (e.g. Chromium, Google Chrome, or Chrome Edge) so `isChromium` more accurately reflects what is being done. |
```